src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -92,19 +92,27 @@
                                     <aura:if isTrue="{!equals(v.selectedDonation, '')}">
                                         <p>{!$Label.c.bgeFormNewDonationHelptext}</p>
                                         <aura:set attribute="else">
-                                            <p class="slds-m-right_x-small">{!$Label.c.bgeFormUpdateHelptext}
-                                                <lightning:formattedUrl value="{! '/' + v.selectedDonation.Id}"
-                                                                        label="{!v.selectedDonation.Name}"
-                                                                        class="slds-m-left_x-small"
-                                                                        target="_blank"/>
-                                            </p>
+                                        <p class="slds-m-right_x-small">
+                                            <aura:if isTrue="{!equals(v.selectedDonation.applyPayment, true)}">
```

Comment:
Thoughts on condensing this down to isTrue="{!v.selectedDonation.applyPayment}"?

Reply:
Yeah, I like that change, but there are 4 usages of equals() in this component file.  Do you think we should switch them all?

Reply:
updated!

Reply:
I think the other ones are ok, since they need to be evaluated into truthy values, but since this one is a boolean I was thinking it was a bit redundant. Thanks!